test/Microsoft.ApplicationInsights.AspNetCore.Tests/TelemetryInitializers/OperationNameTelemetryInitializerTests.cs

```diff
@@ -66,13 +66,15 @@ public void InitializeDoesNotOverrideOperationNameProvidedInline()
         public void InitializeSetsTelemetryOperationNameToMethodAndPath()
         {
             var contextAccessor = HttpContextAccessorHelper.CreateHttpContextAccessor(new RequestTelemetry(), null);
+            string originalTraceIdentifier = contextAccessor.HttpContext.TraceIdentifier;
 
             var initializer = new OperationNameTelemetryInitializer(contextAccessor);
 
             var telemetry = new EventTelemetry();
             initializer.Initialize(telemetry);
 
             Assert.Equal("GET /Test", telemetry.Context.Operation.Name);
+            Assert.Equal(originalTraceIdentifier, contextAccessor.HttpContext.TraceIdentifier);
```

Comment:
You read out the string value from contextAccessor.HttpContext.TraceIdentifier and then assert that it matches reading it out again?  I don't see this test making a call which would cause the diagnostic listener code to run so asserting that the value of contextAccessor.HttpContext.TraceIdentifier matches itself when nothing has written to it doesn't seem to be very useful.